modules/creating-a-github-application-in-administrator-perspective.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: PROCEDURE
+[id="creating-a-github-application-in-administrator-perspective_{context}"]
+
+= Creating a GitHub application in administrator perspective
+
+As a cluster administrator, you can configure your GitHub application with the OpenShift Container Platform cluster to use pipeline as code. This allows you to execute a set of tasks required for build deployment.
+
+.Prerequisites
+You have installed the Red Hat OpenShift pipelines 1.8 operator from the custom catalog source
+
+.Procedure
+. In the administrator perspective, navigate to *Pipelines* using the left navigation pane.
+. Click *Setup GitHub App* on the right side of the Pipelines page.
+. Type the GitHub application name, for example, pipelines-ci-clustername-ss-test.
+. Click the *Setup* button.
+. Enter your Git password when prompted in the browser.
+. Click *Create GitHub App for `_<username>_`*. The variable `_<username>_` denotes your actual user name.
+
+.Verification
+When the Git application is created, you are navigated back to the OpenShift console that notifies successful setup of the GitHub application and provides all the details of the application.
+
+image::Github-app-details.png[]
+
+The details of the created application are saved as a secret in the OpenShift pipelines namespace.
```

Comment:
```suggestion
The details of the created application are saved as a secret in the `openShift-pipelines` namespace.
```

Reply:
@karthikjeeyar, Incorporated this suggestion.
Please verify.